File: MODUL 3 PEMROGRAMAN 1/PRAK304-2310817120002-SITI RATNA DWINTA SARI.py
for i in range(5):
    a = int(input())
    if a == 0:
        print("Nol")
    elif 1 <= a <= 9:
        print("Satuan")
    elif 10 <= a <= 19:
        print("Belasan")
    elif 20 <= a < 100:
        print("Puluhan")
    else:
        print("Anda Menginput Melebihi Limit Bilangan")

File: MODUL 3 PEMROGRAMAN 1/PRAK304-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main (){
    int a;
    for (int i = 0; i < 5; i++){
        scanf("%d",&a);
    if (a == 0){
        printf("Nol\n"); 
    }else if (a >= 1 && a  <= 9){
        printf("Satuan\n");
    }else if (a >= 10  && a <= 19){
        printf("Belasan\n");
    }else if (a >= 20 && a < 100){
        printf("Puluhan\n");
    }else {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
    } return 0;
}